backend/core/settings.py

```diff
@@ -48,6 +51,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
corsheader 미들웨어는 가능한 제일 위에 위치해야 합니다

참고: https://pypi.org/project/django-cors-headers/
```
CorsMiddleware should be placed as high as possible, especially before any middleware that can generate responses such as Django’s CommonMiddleware or Whitenoise’s WhiteNoiseMiddleware. If it is not before, it will not be able to add the CORS headers to these responses.
```

Reply:
해당내용은 이야기해주신대로 수정완료했습니다.